### ISSUE TITLE
False handle, when field label or heading text starts with a number

### ISSUE THREAD

**Issue by SanboKyo** [verbb/formie#422]
When field label or heading text starts with a number, the handle starts also with a number, which is expectedly returned with an error message.

The error message is displayed incorrectly or the entire form builder.

**Additional info**
- Plugin version: 1.3.26
- Craft version: 3.6.11.2
- Multi-site: Yes 

![Bildschirmfoto 2021-04-06 um 20 29 54](https://user-images.githubusercontent.com/26556339/113762959-6c301880-9719-11eb-9faf-237a1d1fc343.jpg)

![Bildschirmfoto 2021-04-06 um 20 30 24](https://user-images.githubusercontent.com/26556339/113762919-620e1a00-9719-11eb-88d9-b32c590316c2.jpg)

![Bildschirmfoto 2021-04-06 um 20 30 36](https://user-images.githubusercontent.com/26556339/113762948-69352800-9719-11eb-8f0c-93a92041ceff.jpg)

**Comment by engram-design** [comment]
I'll look at ways to handle this, but:

> The error message is displayed incorrectly or the entire form builder.

Can you explain what you mean about this? The error message seems to be displayed correctly to me?

**Comment by SanboKyo** [comment]
I create a new form with some fields and headings:

![Bildschirmfoto 2021-04-07 um 10 47 07](https://user-images.githubusercontent.com/26556339/113838589-3299f480-978f-11eb-832b-158584523089.jpg)

Click on Save and get this:

![Bildschirmfoto 2021-04-07 um 10 47 29](https://user-images.githubusercontent.com/26556339/113838749-5eb57580-978f-11eb-9188-b772e0e97b30.jpg)

But there are some more bugs ... If I delete a heading now:

![Bildschirmfoto 2021-04-07 um 10 48 20](https://user-images.githubusercontent.com/26556339/113839090-a9cf8880-978f-11eb-8ef2-cd1b2fededf9.jpg)

The entire form is deleted! Before form is saved:

![Bildschirmfoto 2021-04-07 um 10 48 31](https://user-images.githubusercontent.com/26556339/113839129-b358f080-978f-11eb-8941-f34548eb4981.jpg)

... and after:

![Bildschirmfoto 2021-04-07 um 10 49 35](https://user-images.githubusercontent.com/26556339/113839488-029f2100-9790-11eb-87a7-584806964e17.jpg)

**Comment by engram-design** [comment]
Thanks for clarifying, I'll see what's going on there. The handle generation issue (with the number first) should be fixed in https://github.com/verbb/formie/commit/18908ac47dac64f0f1b3ddc1fff0390bb801920d

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.26",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.27](https://github.com/verbb/formie/releases/tag/1.3.27)
